src/coord/config.c

```diff
@@ -136,7 +148,8 @@ static RSConfigOptions clusterOptions_g = {
             {.name = "OSS_GLOBAL_PASSWORD",
              .helpText = "Global oss cluster password that will be used to connect to other shards",
              .setValue = setGlobalPass,
-             .getValue = getGlobalPass},
+             .getValue = getGlobalPass,
+             .flags = RSCONFIGVAR_F_IMMUTABLE},
```

Comment:
Is this a breaking change?
Is it only breaking the ability to set the password, but actually it was only considered in module init?

Reply:
Effectively not, since we rely on load-time setting of this config param only